cnxarchive/views.py

```diff
@@ -886,6 +897,7 @@ def sitemap(request):
                             '[^0-9a-z]+', ' ', 'g')), ' +', '-', 'g'),
                         revised
                     FROM latest_modules
+                    WHERE portal_type != 'CompositeModule'
```

Comment:
Any reason not to put the CompositeModule in the sitemap?

Perhaps we should address this later? If so, let's note it somewhere.